app/resources/documentation/manual/index.md

```diff
@@ -843,7 +843,9 @@ In TrenchBroom, there is the notion of a current texture, which we have already
 
 To change the texture of the currently selected faces, left click on a texture in the texture browser. This also works if you have selected brushes (and nothing else) - in this case, the new texture is applied to all faces of the currently selected brushes. 
 
-To transfer the texture and texture attributes (such as the offset or scale) from one face to another, start by selecting the source face with #key(Shift) + left click. Then, hold one of the following modifier combinations depending on what you want to transfer:
+You can also transfer texture and attributes from one face to another. "Attributes" in this context refers to almost any characteristic &mdash; such as offset, scale, or surface flags &mdash; that you can modify through the [face attribute editor](#face_attribute_editor). The sole exception to this is content flags; the content flags on the target face(s) will always be preserved unchanged.
```

Comment:
I think the second sentence is missing a word: "almost any characteristic ... "

Reply:
I could be having a senior moment but I think that sentence is OK? Maybe the mdash entities are obfuscating things... here's what the rendered markdown looks like for comparison:

![manual](https://user-images.githubusercontent.com/2068148/82835727-ba2d8c00-9e79-11ea-82e1-639638422ccf.png)

Reply:
Looks fine as is.

Reply:
Ok, now I get it, it's fine!